engine-tests/build.gradle

```
@@ -107,6 +107,8 @@ task unitTest(type: Test) {
         excludeTags "MteTest", "TteTest"
     }
     systemProperty("junit.jupiter.execution.timeout.default", "1m")
+
+    jacoco.excludes = ["org.terasology.protobuf.*", "*MethodAccess", "*FieldAccess"]
```

Should we add a comment here to reference the other location, or at least ask to search for `jacoco.excludes` whenever this line is edited? 

Likewise in `common.gradle`.

```suggestion
    // Keep in sync with other exclude-lists for Jacoco, e.g., in 'common.gradle' 
    jacoco.excludes = ["org.terasology.protobuf.*", "*MethodAccess", "*FieldAccess"]
```